tests/app/delivery/test_send_to_providers.py

```diff
@@ -393,6 +393,23 @@ def test_send_email_should_use_service_reply_to_email(sample_service, sample_ema
     )
 
 
+@pytest.mark.parametrize("service_fixture", ["sample_service", "sample_service_with_email_branding"])
+def test_send_email_works_with_and_without_email_branding(request, service_fixture, sample_email_template, mocker):
+    service = request.getfixturevalue(service_fixture)
+    mocker.patch("app.aws_ses_client.send_email", return_value="reference")
+
+    db_notification = create_notification(template=sample_email_template, reply_to_text="foo@bar.com")
+    create_reply_to_email(service=service, email_address="foo@bar.com")
```

Comment:
The tests pass without this line, so I don't think we need to create the reply_to email address for the test

Reply:
Patched in fixup commit 5da28b6b9ed665cef0dbe809bb83c286ea0c9b43